### ISSUE TITLE
loss function?

### ISSUE THREAD

**Issue by yuxuan11** [pjreddie/darknet#854]
![loss](https://user-images.githubusercontent.com/25193958/40957319-19b842ea-68c7-11e8-9371-dff348641be9.jpg)

I want to ask you a question，what is Ci? Is the category number of the object?

**Comment by Kranberry** [comment]
Ci is the confidence score. Each bounding box predicts a confidence score, along with 4 coordinates. i is the index of the grid cell, and theoretically there should be a confidence score for each bounding box, not each grid cell. But in the expression above, only one bounding box will be considered for each grid cell during training ( the bounding box which has the highest IOU with the ground truth box). So since only one bounding box, only one confidence score per grid cell. Thus, that's what Ci represents.
